cumulus/polkadot-parachain/Cargo.toml

```
@@ -161,3 +161,6 @@ try-runtime = [
 	"shell-runtime/try-runtime",
 	"sp-runtime/try-runtime",
 ]
+fast-runtime = [
+	"bridge-hub-rococo-runtime/fast-runtime",
+]
```

```suggestion
fast-runtime = []
```
I think it should work just like this (without defaulting `"bridge-hub-rococo-runtime/fast-runtime"`),
@alistair-singh @claravanstaden could you, please, try it if it works for you local testing?

---

@bkontur it doesn't work, it seems. The `fast-runtime` feature should be propagated to the bridge hub rococo runtime for it to work.